src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -1175,7 +1175,7 @@ public function delete()
      *
      * This method protects developers from running forceDelete when trait is missing.
      *
-     * @return void
+     * @return bool|null
```

Comment:
Should comply `Model::delete()` however `SoftDeletes::forceDelete()` does not. If this merged, SoftDeletes should be fixed either.